### PR TITLE
Fix blueberry

### PR DESCRIPTION
**Pull request by crobibero**
Should fix https://github.com/jellyfin/jellyfin/issues/4744

cc @anthonylavado as I don't have any way to test this.

--
ImageFilter was added in https://github.com/jellyfin/jellyfin/pull/3772. I'm not sure why it was added but it seems to be the source of the blueberries.

### CODE REVIEW

**Review by BaronGreenback — APPROVED** [review]

**Comment by anthonylavado** [comment]
This did not fix the issue. Back to the drawing board, I guess.

**Comment by crobibero** [comment]
This was a first guess, I’ll have to read the linked issue more in depth

**Comment by anthonylavado** [comment]
@crobibero Best I can guess is that the issue is happening specifically with any resizing. It doesn't happen if the image is already at the correct size.

**Comment by frankyifei** [comment]
The best way was stated in upstream bug report and it is not a bug

mattleibow commented on 9 Sep
This is most probably because you have a new bitmap using the "platform default" which is BGR on Windows and RGB on macOS:

var bitmap = new SKBitmap(pixelWidth, pixelHeight);
If you are wanting to make sure the color orders are as expected, use the overload that takes a SKColorType and you can control exactly what order they are.

Or, you can check the SKImageInfo.PlatformColorType and check to see if it is Bgra8888 or Rgba8888

Or, you can use SKSwizzle so switch bytes.

The best is to o with the first option as there are no checks and no different logic and no extra work.

**Comment by anthonylavado** [comment]
This does indeed fix the issue for me. Waiting for the review from EraYaN because the filter code was added in his commit.

**Comment by anthonylavado** [comment]
This also still fixes it for me. Still waiting on other reviewer.

**Review by nyanmisaka — APPROVED** [review]

**Comment by joshuaboniface** [comment]
If this still requires more work, please make a separate PR for that :+1:.

**Pull request merged**
